### PR TITLE
[Bf16 & int8] [example & ckprofiler]

### PR DESCRIPTION
**Pull request by rocking5566**

### CODE REVIEW

**Comment by asroy** [comment]
Please add unit test under ```test/```

File: device_operation/src/device_conv2d_fwd_xdl_nhwc_kyxc_nhwk_bf16_instance.cpp [hunk 1]
```diff
@@ -32,19 +32,19 @@ using device_conv2d_fwd_xdl_nhwc_kyxc_nhwk_bf16_instances = std::tuple<
...
-        DeviceConv2dFwdXdl_Input_N_Hi_Wi_C_Weight_K_Y_X_C_Output_N_Ho_Wo_K<  ushort,  ushort,  ushort,     F32, PassThrough, PassThrough, PassThrough, ConvFwdDefault,    64,    32,    64,     4,  8,   32,   32,    1,    2,     S<4, 16, 1>,     S<1, 0, 2>,     S<1, 0, 2>,              2,              8,              8,      true,     S<4, 16, 1>,     S<1, 0, 2>,     S<1, 0, 2>,             2,              8,              8,      true,               7,               1>
+        DeviceConv2dFwdXdl_Input_N_Hi_Wi_C_Weight_K_Y_X_C_Output_N_Ho_Wo_K<  bhalf_t,  bhalf_t,  bhalf_t,     F32, PassThrough, PassThrough, PassThrough, ConvFwdDefault,   256,   256,   128,     4,  8,   32,   32,    4,    2,     S<4, 64, 1>,     S<1, 0, 2>,     S<1, 0, 2>,              2,              8,              8,      true,     S<4, 64, 1>,     S<1, 0, 2>,     S<1, 0, 2>,             2,              8,              8,      true,               7,               1>,
```
Comment (asroy) [inline device_operation/src/device_conv2d_fwd_xdl_nhwc_kyxc_nhwk_bf16_instance.cpp:35]: Please fix all alignment issue in "instances.cpp"

image

**Review by asroy — CHANGES_REQUESTED** [review]
Please add test and fix alignment issue

**Comment by rocking5566** [comment]
> Please add test and fix alignment issue

Done

**Review by asroy — DISMISSED** [review]

**Comment by asroy** [comment]
Please fix merge conflict and CI failure

**Review by asroy — APPROVED** [review]

**Pull request merged**